tests/installation/partitioning_warnings.pm

```diff
@@ -19,40 +19,92 @@ use testapi;
 use partition_setup;
 use version_utils 'is_storage_ng';
 
+sub process_warning {
+    my (%args) = @_;
+    assert_screen 'partition-warning-' . $args{warning};
+    $args{key} //= 'alt-y';    # By default press yes, to accept the settings
+    wait_screen_change { send_key $args{key} };
+}
+
+sub process_missing_special_partitions {
+    # Have missing boot/zipl partition warning only in storage_ng
+    if (is_storage_ng && check_var('ARCH', 's390x')) {    # s390x needs /boot/zipl on ext partition
+        process_warning(warning => 'no-boot-zipl', key => 'alt-n');
```

Comment:
from this PR looks like alt-n should be default option , I see here more calls with alt-n then with alt-y. But feel free to ignore this comment

Reply:
That's true. I will leave it for now, only because by default we proceed with the installation acknowledging that we know what we are doing and with No, it will bring us back to expert partitioner.